mappings/net/minecraft/client/util/math/MatrixStack.mapping

```diff
@@ -36,6 +36,17 @@ CLASS net/minecraft/unmapped/C_cnszsxvd net/minecraft/client/util/math/MatrixSta
 			COMMENT the translation amount on the Y axis
 		ARG 3 z
 			COMMENT the translation amount on the Z axis
+	METHOD m_tweczbqs rotateAround (Lorg/joml/Quaternionf;FFF)V
+		COMMENT Applies the rotation transformation of the given {@link Quaternionf} to the current matrix
+		COMMENT while using {@code (oX, oY, oZ)} as the rotation origin.
+		ARG 1 quaternion
+			COMMENT the rotation transformation quaternion
+		ARG 2 oX
+			COMMENT the X coordinate of the rotation origin
+		ARG 3 oY
+			COMMENT the Y coordinate of the rotation origin
+		ARG 4 oZ
+			COMMENT the Z coordinate of the rotation origin
```

Comment:
I don't think the 'origin' `o` is needed for these parameters

Reply:
They might not be strictly needed, but in this context I find it conveying more meaning than just x, y, z.

Reply:
I'm with martrix on this one, if you told me "rotate around x" I'd know what you meant instantly but if the command was "rotate around oX" it would take me a second to decipher it